clinical_refs/management/commands/seed_refs_diabetes.py

```
@@ -0,0 +1,39 @@
+        "url": "https://example.org/ada/2025/pharmacologic-therapy",
+        "topic": "diabetes",
+        "section": "therapy",
+        "lang": "en",
+        "tags": ["metformin","sglt2","glp1"],
+    },
+]
+
+class Command(BaseCommand):
+    help = "Seed ADA 2025 clinical references for diabetes"
+
+    def handle(self, *args, **options):
```

Ruff (0.12.2)

Missing return type annotation for public function `handle`

Add return type annotation: `None`

(ANN201)